pkg/controller/master/image/backing_image_controller.go

```diff
@@ -56,12 +67,25 @@ func (h *backingImageHandler) OnChanged(_ string, backingImage *lhv1beta2.Backin
 	// for that new field.  Another, simpler, alternative would be to just
 	// drop the ImageImported.IsUnknown check entirely, and let the following
 	// loop run through on every OnChanged event.
-	if !harvesterv1beta1.ImageImported.IsUnknown(vmImage) && vmImage.Status.VirtualSize == backingImage.Status.VirtualSize {
+
+	// Found LH restruct DiskFileStatusMap in https://github.com/harvester/harvester/issues/6894,
+	// if Harvester finds the restruct, it should keep sync with LH backing image
+	if !harvesterv1beta1.ImageImported.IsUnknown(vmImage) && vmImage.Status.VirtualSize == backingImage.Status.VirtualSize &&
+		backingImageHasReadyDisk(backingImage) {
 		return nil, nil
 	}
+
+	// Todo: LH has bi HA from v1.7, Harvester should consider have the file map structure
```

Comment:
Do we have an issue for this ? If so, perhaps we can add github issue link here.

Reply:
Issue link https://github.com/harvester/harvester/issues/6936, thanks.